Assets/Scripts/Managers/InstanceManager.cs

```diff
@@ -100,11 +100,11 @@ public static class InstanceManager {
         }
 
         public static GameObject DisplayFullscreen(string name) {
-            return Instantiate(name, Canvas.transform.Find("Fullscreen"));
+            return Instantiate(name, Canvas.transform.Find("Player UI").Find("Fullscreen"));
         }
 
         public static GameObject DisplayWindow(string name) {
-            return Instantiate(name, Canvas.transform.Find("Windows Section"));
+            return Instantiate(name, Canvas.transform.Find("Player UI").Find("View Area").Find("Middle Section").Find("Windows Section"));
```

Comment:
Could we not store a reference for this instead of instantiating it every time?

Reply:
Do you mean a serialized field?

Reply:
I mean a field in the class for it e.g:

```
GameObject windowsSection;

public static GameObject DisplayWindow(string name) {
    if (windowsSection == null ) { 
        windowsSection = Instantiate(name, Canvas.transform.Find("Player UI").Find("View Area").Find("Middle Section").Find("Windows Section"));
    }

    return windowsSection;
}
````

Reply:
> 
> 
> I mean a field in the class for it e.g:
> 
> ```
> GameObject windowsSection;
> 
> public static GameObject DisplayWindow(string name) {
>     if (windowsSection == null ) { 
>         windowsSection = Instantiate(name, Canvas.transform.Find("Player UI").Find("View Area").Find("Middle Section").Find("Windows Section"));
>     }
> 
>     return windowsSection;
> }
> ```

Sure, ill implement that now

Reply:
Just another question, is this GameObject then destroyed later?

Reply:
What GameObject?

Reply:
The return value of `DisplayWindow`

Reply:
Depends on the object, so currently, this is currently just used to Instantiate a task, which destroys itself when completed or closed using the X in the top right.

Reply:
Also those gameobjects are now referenced on sceneload